tests/behaviour/concept/serialization/json/BUILD

```diff
@@ -48,6 +48,8 @@ typedb_behaviour_py_test(
         "//tests/behaviour/util:util",
         "//tests/behaviour/config:parameters",
         "//tests/behaviour/background",
+        "@vaticle_typedb_driver_java//rust:typedb_client_python",
+        "//:python-lib",
```

Comment:
Why are we depending on both? Shouldn't this just be `//:python-lib`?

Reply:
Yes, it should.